components/sup/Cargo.toml

```diff
@@ -28,6 +28,7 @@ habitat_core = { path = "../core" }
 habitat_depot_client = { path = "../builder-depot-client" }
 habitat-eventsrv-client = { path = "../eventsrv-client" }
 habitat-launcher-client = { path = "../launcher-client" }
+habitat-launcher-protocol = { path = "../launcher-protocol" }
```

Comment:
ok. I had debated that option too and went with this in an effort to keep the const in one place. Will update.